[Ready for Review] - Adding a concept TOC for Create prerequisites (#126069)
Azure/azure-content: articles/governance/machine-configuration/concepts/prerequisites.md

---
title: Prerequisites to use Machine Configuration Policies
description: Prerequisites needed to use Machine Configuration policies.
ms.date: 03/04/2025
ms.topic: conceptual
---
# Prerequisites to use Machine Configuration Policies
To ensure that Machine Configuration policies can be applied successfully, certain prerequisites must be met. This document outlines the necessary steps and considerations for deploying these prerequisites.
The following prerequisites are needed for Machine Configuration policies to work.
  
- Machine Configuration Extension   
- Enable MSI/User assigned identity

Machine Configuration provides an initiative which will help to achieve all the above prerequisites.

> Please assign this built-in initiative at the scope you are planning to use Machine Configuration policies - *Deploy prerequisites to enable Guest Configuration policies on virtual machines*

Reviewer: ```suggestion
---

# Prerequisites to use Machine Configuration policies

Using Machine Configuration policies requires a machine to meet certain prerequisites. This
document outlines the necessary steps and considerations for deploying these prerequisites.

To use Machine Configuration policies, take the following actions for each VM you want to manage:

- Install the Machine Configuration Extension.
- Enable MSI/User assigned identity

To resolve these prerequisites, create the following assignment with Azure Policy:

- [Deploy prerequisites to enable Guest Configuration policies on virtual machines][01].

<!-- Link reference definitions -->
[01]: https://github.com/Azure/azure-policy/blob/master/built-in-policies/policySetDefinitions/Guest%20Configuration/Prerequisites.json
```

Author: Pushed a new commit.

Author: One of the problems we are struggling with Machine Configuration is, customers are not aware of the pre-requisites policy requirement for Machine Configuration policies to work.
We can also reference the article you suggested for more details, but in my opinion having an explicit mention about pre-requisites would help. Not an expert in this area so I will trust you more on this :)

> # Prerequisites to use Machine Configuration Policies

Reviewer: ```suggestion
---
title: Prerequisites to use Machine Configuration Policies
description: Describes steps required for using Machine Configuration.
ms.date: 04/01/2025
ms.topic: conceptual
---

# Prerequisites to use Machine Configuration Policies
```

> 	To ensure that Machine Configuration policies can be applied successfully, certain prerequisites must be met. This document outlines the necessary steps and considerations for deploying these prerequisites.
> 	The following prerequisites are needed for Machine Configuration policies to work.

Reviewer: ```suggestion
To ensure that Machine Configuration policies can be applied successfully, certain prerequisites
must be met. This document outlines the necessary steps and considerations for deploying these
prerequisites.

The following prerequisites are needed for Machine Configuration policies to work:
```

	  
	- Machine Configuration Extension   
	- Enable MSI/User assigned identity
	

> 	Machine Configuration provides an initiative which will help to achieve all the above prerequisites.
> 	Please assign this built-in initiative at the scope you are planning to use Machine Configuration policies - *Deploy prerequisites to enable Guest Configuration policies on virtual machines*

Reviewer: ```suggestion
To automatically fulfill these prerequisites, you can use the **Deploy prerequisites to enable
Guest Configuration policies on virtual machines** built-in initiative. Assign the initiative at
the scope you're planning to use Machine Configuration policies.
```
